pennylane/plugins/default_gaussian.py

```diff
@@ -814,6 +814,16 @@ def expand(self, S, wires):
         return S2
 
     def expval(self, observable, wires, par):
+        r"""Expectation value of observable on specified wires.
+
+        Args:
+          observable      (str): name of the observable
+          wires (Sequence[int]): target subsystems
+          par    (tuple[float]): parameter values
+
+        Returns:
+          float: expectation value :math:`\expect{A} = \bra{\psi}A\ket{\psi}`
+            """
```

Comment:
These are inherited methods, so the docstring should not be filled out - instead, Sphinx will automatically use the base class docstring in `DefaultGaussian`.

Reply:
I know, but I didn't like the inherited docstrings. It was showing extra lines about "For plugin developers, blah blah blah" which I thought were redundant